conf/standalone.conf

```diff
@@ -268,3 +268,14 @@ keepAliveIntervalSeconds=30
 
 # How often broker checks for inactive topics to be deleted (topics with no subscriptions and no one connected)
 brokerServicePurgeInactiveFrequencyInSeconds=60
+
+### --- WebSocket --- ###
+
+# Enable the WebSocket API service in broker
+webSocketServiceEnabled=false
```

Comment:
Web socket is already enabled by default in standalone mode (it's configured directly in the standalone starter.. probably not the best idea though)